### ISSUE TITLE
move `<footer>` outside of `<main>`

### ISSUE THREAD

**Issue by sukiletxe** [getnikola/nikola#2309]
Imo the footer of a page is not part of the main content of a site (just like for example a menu). Thus, the last `<footer>` element should be moved outside and below `<main>` in the default themes for accessibility purposes.

**Comment by ralsina** [comment]
I tend to agree but am far from an expert. Should be easy-ish?

**Comment by sukiletxe** [comment]
Assuming that the css will not get ugly as a result of this change 
(sorry, I cannot test that, but I think it won't), it would just be 
editing base.tmpl in bootstrap3. Just put the closing </div> of 
role='main' before the <footer> (and jingify the result, of course). 
Base already does this correctly.
As a side note: Why do you use <main> in base and <div role = 'main' in 
bootstrap3?

**Comment by Kwpolska** [comment]
> As a side note: Why do you use <main> in base and <div role = 'main' in bootstrap3?

That’s a question for [Bootstrap devs](https://getbootstrap.com/examples/jumbotron/).

**Comment by Kwpolska** [comment]
Closing as this generally isn’t our bug, and let’s not break our base theme either.

**Comment by sukiletxe** [comment]
Sorry for being so tedious, but I've just found ot that there wouldn't be any problem of styling or anything if we just assigned the `role="main"` to the inner `div` (`.body-content`) instead of the `#content` `div`. The outer one (`#content`) just contains `.body-content` and the footer.

**Comment by sukiletxe** [comment]
Actually, no, the inner `div` is closed after the footer. Does it have any function? I can see that the `.body-content` class is not referenced in any css. Can it be messed around with? Does it have any styling function?

**Comment by ralsina** [comment]
Honestly, I don't think anyone remembers exactly why or how each div got to be like it is.

If this is in the bootstrap-based theme, then it's that way because that's how bootstrap did it in the sample pages.
